package.json

```diff
@@ -76,11 +77,12 @@
     "rollup-plugin-url": "^2.2.2",
     "source-map-explorer": "2.0.1",
     "start-server-and-test": "^1.10.0",
+    "tsconfig-paths-webpack-plugin": "^3.2.0",
     "typescript": "^3.5.3",
     "webpack": "4.39.1"
   },
   "scripts": {
-    "lint": "eslint '{packages,examples}/**/*.{js,jsx,ts,tsx}' --ignore-pattern node_modules --quiet",
+    "lint": "eslint '{packages,examples}/**/*.{js,jsx,ts,tsx}' --ignore-pattern node_modules --quiet && madge --circular packages/discovery-react-components/src/*",
```

Comment:
I think it would be nice to have this as separate methods like 
```
"lint": "yarn run eslint && yarn run circular"

Reply:
can we define a `lint` command in our `discovery-react-components` package? i think running `yarn lint` will automatically run that as well